File: example/scalalib/testing/6-test-group-stealing/foo/test/src/GroupY3.scala
package foo
import utest._
object GroupY3 extends RandomTestsUtils {
  def tests = Tests {
    test("test1") { testGreeting("Ra", 21) }
    test("test2") { testGreeting("Saturn", 18) }
    test("test3") { testGreeting("Thor", 19) }
    test("test4") { testGreeting("Ullr", 22) }
    test("test5") { testGreeting("Venus", 17) }
  }
} 

File: example/kotlinlib/testing/6-test-group-stealing/foo/test/src/foo/GroupY3.kt
package foo

import org.junit.jupiter.api.Test

class GroupY3 : RandomTestsUtils() {
    @Test
    @Throws(Exception::class)
    fun test1() { testGreeting("Ra", 21) }
    
    @Test
    @Throws(Exception::class)
    fun test2() { testGreeting("Saturn", 18) }
    
    @Test
    @Throws(Exception::class)
    fun test3() { testGreeting("Thor", 19) }
    
    @Test
    @Throws(Exception::class)
    fun test4() { testGreeting("Ullr", 22) }
    
    @Test
    @Throws(Exception::class)
    fun test5() { testGreeting("Venus", 17) }
} 